src/services/utils.ts

```diff
@@ -60,59 +67,44 @@ const downloadFile = (blob: Blob, filename: string, type?: FileType) => {
     document.body.removeChild(link);
 };
 
-const prepareRequest = (init: any, token?: string) => {
+function prepareRequest(init?: InitRequest, token?: Token): RequestInit {
```

Comment:
If we start to type the Token, it need to be changed everywhere

Reply:
That's the idea

Reply:
Done